AdminWebsite/AdminWebsite/Services/HearingsService.cs

```diff
@@ -511,10 +511,7 @@ public async Task<TeleConferenceDetails> GetTelephoneConferenceDetails(Guid hear
             // Judge and panel member is manually created in AD, no need to create one
             var ejudFeatureFlag = await _bookingsApiClient.GetFeatureFlagAsync(nameof(FeatureFlags.EJudFeature));
```

Comment:
Can remove the call to the bookingsApiClient, to get the Ejud flag, as it's not being used in the if (condition on 514 anymore)